hnn_core/parallel_backends.py

```diff
@@ -76,7 +76,7 @@ def __enter__(self):
 
         return self
 
-    def __exit__(self, type, value, traceback):
+    def __exit__(self, feed_type, value, traceback):
```

Comment:
here I think the original `type` was intended, that is the Python keyword. Although I'm not sure why it is passed since it's not used down below. @blakecaldwell can we remove the arguments after `self`?

Reply:
What i had to do in this?

• Remove 'feed_type' like this:-
def __exit__(self,value,traceback): 

OR SOMETHING ELSE...?

Reply:
just do:

```suggestion
    def __exit__(self):
```

I hope nothing breaks